multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -658,12 +729,119 @@ def get_coco_format_classes(sample_data_path):
     return [cat["name"] for cat in annotation["categories"]]
 
 
-def get_voc_format_classes(sample_data_path):
-    return NotImplementedError
+def get_voc_format_classes(root):
+    if is_url(root):
+        root = download(root)
+    rpath = Path(root).expanduser()
+
+    labels_file = os.path.join(rpath, "labels.txt")
+    if os.path.exists(labels_file):
+        with open(labels_file) as f:
+            class_names = [line.rstrip().lower() for line in f]
+        print(f"using class_names in labels.txt: {class_names}")
+    else:
+        logger.warning(
```

Comment:
if labels.txt does not exist, we should dump that file

Reply:
Fixed in the new commit 103b67d4248539fc4a29b9001ca1c714d5080c20